pkgs/applications/networking/mailreaders/claws-mail/default.nix

```diff
@@ -80,8 +82,16 @@ stdenv.mkDerivation {
     ++ optional (!enablePluginVcalendar) "--disable-vcalendar-plugin"
     ++ optional (!enableSpellcheck) "--disable-enchant";
 
+  enableParallelBuilding = true;
+
   postInstall = ''
     mkdir -p $out/share/applications
     cp claws-mail.desktop $out/share/applications
+
+    if test -e $out/share/mime; then
```

Comment:
Can you explain this better? If the build is always the same, in which case does it cause this error?